src/openrct2/actions/TrackPlaceAction.hpp

```diff
@@ -315,7 +315,7 @@ DEFINE_GAME_ACTION(TrackPlaceAction, GAME_COMMAND_PLACE_TRACK, TrackPlaceActionR
                 if (surfaceElement == nullptr)
                     return std::make_unique<TrackPlaceActionResult>(GA_ERROR::UNKNOWN, STR_NONE);
 
-                uint8_t waterHeight = surfaceElement->GetWaterHeight() * 2;
+                uint8_t waterHeight = surfaceElement->GetWaterHeight() / COORDS_Z_STEP;
```

Comment:
Make a note for the future to refactor TrackPlaceAction to remove *8 /8 (COORDS_Z_STEP) from this function.